components/net/bluetooth_thread.rs

```diff
@@ -763,7 +755,7 @@ impl BluetoothManager {
             value = self.get_gatt_descriptor(&mut adapter, &id)
                         .map(|d| d.read_value().unwrap_or(vec![]));
         }
-        let _ = sender.send(value.ok_or(String::from(VALUE_ERROR)));
+        let _ = sender.send(value.ok_or(BluetoothError::NotFound));
```

Comment:
This is described here: https://webbluetoothcg.github.io/web-bluetooth/#error-handling, but this should be `NotSupportedError`.